scripts/check_workspace_inherited_default_features.py

```
@@ -0,0 +1,95 @@
+
+try:
+    import tomllib
+except ModuleNotFoundError:  # pragma: no cover
+    try:
+        import tomli as tomllib  # type: ignore[no-redef]
+    except ModuleNotFoundError:
+        print(
+            "error: Python 3.11+ or the 'tomli' package is required to read TOML",
+            file=sys.stderr,
+        )
+        raise SystemExit(2)
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)